woocommerce/woocommerce-gateway-stripe
# Add "Connect Stripe Account" UI

Fixes #1757

### Description
If  `_wcstripe_feature_upe_settings` is enabled and a Stripe account is not connected a `card` containing the related information and `Connect` button appears in the main Stripe settings page as well as in other non-UPE legacy payment method pages.

### Testing instructions

- Disconnect Stripe account (clear all the keys from old UI)
- Enable the `_wcstripe_feature_upe_settings` flag
- Check the main Stripe settings page and other legacy payment method settings pages

Desktop view of the Stripe settings page
![main stripe page](https://user-images.githubusercontent.com/33387139/131618036-0faf89b9-0f26-448a-a178-8c4718eee5a6.png)

Mobile view of the Stripe settings page
![mobile view stripe settings](https://user-images.githubusercontent.com/33387139/131618061-0f014184-27fa-4644-b875-6ca91938fd02.png)

Desktop view of the other methods' settings page
![legacy page](https://user-images.githubusercontent.com/33387139/131618084-f3d79ada-f413-40e3-8b67-b54f867122c0.png)

### QA checklists

- [ ] If  `_wcstripe_feature_upe_settings`  is disabled the old settings page is visible in Stripe and other payment method settings pages
- [ ] If  `_wcstripe_feature_upe_settings`  is enabled and Stripe is not connected, the `Connect Stripe Account` card is visible in Stripe settings page
- [ ]  If  `_wcstripe_feature_upe_settings`  is enabled and Stripe is not connected, the `Connect Stripe Account` card is visible in  all legacy method settings pages
- [ ]  If  `_wcstripe_feature_upe_settings`  is enabled and Stripe is connected, the `Connect Stripe Account` card should not be present
